ert_gui/model/job_list.py

```diff
@@ -92,8 +92,12 @@ def headerData(
             elif section == 3:
                 return "End time"
             elif section == 4:
-                return "Current Memory Usage"
+                return "STDOUT"
             elif section == 5:
+                return "STDERR"
+            elif section == 6:
+                return "Current Memory Usage"
+            elif section == 7:
                 return "Max Memory Usage"
```

Comment:
_Could_ refactor this such that these are enumerated from `COLUMNS` in the snapshot model package.